cg/apps/housekeeper/hk.py

```diff
@@ -649,3 +653,14 @@ def update_archive_retrieved_at(
         for archive in archives_to_update:
             archive.retrieval_task_id = new_retrieval_job_id
         self._store.session.commit()
+
+    def get_spring_files_retrieved_before(self, date: dt.datetime):
```

Comment:
Change import to `from datetime import datetime`
```suggestion
    def get_spring_files_retrieved_before(self, date: datetime):
```